include/ibamr/IBFESurfaceMethod.h

```diff
@@ -295,13 +311,29 @@ class IBFESurfaceMethod : public IBStrategy
     /*!
      * Interpolate the Eulerian velocity to the curvilinear mesh at the
      * specified time within the current time interval.
+     * \note WSS is computed if the jump(s) in the fluid shear stress is applied.
      */
     void interpolateVelocity(
         int u_data_idx,
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::CoarsenSchedule<NDIM> > >& u_synch_scheds,
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::RefineSchedule<NDIM> > >& u_ghost_fill_scheds,
         double data_time) override;
 
+    /*!
+     * Compute the fluid traction if all jump conditions are applied.
+     */
+    void computeFluidTraction(double current_time, unsigned int part = 0);
+
+    /*!
+     * Compute the pressure used in the calculation of the fluid traction (pressure jump condition needs to be applied).
+     */
+    void interpolatePressureForTraction(int p_data_idx, double data_time, unsigned int part = 0);
```

Comment:
Should this be a `public` function, or should it be a `private` helper function? (Does it make sense for users to call this function, or should it only be called from inside the class?)

If it makes sense for external callers to use these functions, would it be more clear to split them into functions that evaluate the pressure and viscous stresses on the interface (e.g. `evaluatePressureOnInterface()` and `evaluateViscousStressOnInterface()`). `interpolatePressureForTraction()` sounds like this is a helper function.

I would suggest making all of these functions `private` for now, and we can worry about the `public` interface later if needed (e.g. to support better post processing).